admin/app/components/solidus_admin/tax_categories/edit/component.html.erb

```diff
@@ -5,6 +5,11 @@
         <%= render component("ui/forms/field").text_field(f, :name) %>
         <%= render component("ui/forms/field").text_field(f, :tax_code) %>
         <%= render component("ui/forms/field").text_field(f, :description) %>
+        <%= render component("ui/forms/field").select(
```

Comment:
This should as well be behind a feature toggle, in order to not overwhelm and confuse store owners that will never ever use this feature, especially outside of the EU (which are the vast majority of our users).